gnovm/pkg/gnolang/transcribe.go

```diff
@@ -271,7 +271,7 @@ func transcribe(t Transform, ns []Node, ftype TransField, index int, n Node, nc
 		} else {
 			cnn = cnn2.(*FuncLitExpr)
 		}
-		for idx := range cnn.Body {
+		for idx := 0; idx < len(cnn.Body); idx++ { // the len of Body could change when we decompose a statemet to mulitiple statments.
```

Comment:
```suggestion
		for idx := 0; idx < len(cnn.Body); idx++ { // Iterate over Body; its length can change if a statement is decomposed.
```

Spotted a few typos in your PR. Just wanted to make sure you're aware before submitting smaller PRs to the main repo.